Source/WebKit/UIProcess/API/gtk/WebKitWebView.h

```diff
@@ -293,9 +293,7 @@ struct _WebKitWebViewClass {
                                                 WebKitWebProcessTerminationReason reason);
     gboolean   (* user_message_received)       (WebKitWebView               *web_view,
                                                 WebKitUserMessage           *message);
-
-    /*< private >*/
-    void (*_webkit_reserved0) (void);
+    void       (* initialize_web_extensions)   (WebKitWebView               *web_view);
```

Comment:
This is not really needed. Adding a vfunc to the class struct is done when one wants to provide a default implementation and/or to allow overriding it in subclasses. You actually pass `0` as the `class_offset` when creating the signal below, which means this slot is unused.

If you end up adding a signal to `WebKitWebView` you do not need to consume our last padding entry. Even without a slot in the class struct, a subclass can still use [g_signal_override_class_handler()](https://docs.gtk.org/gobject/func.signal_override_class_handler.html) to override the default implementation of an vfunc. It's a bit more involved, but I think it's a reasonable tradeoff for vfuncs which do not need a default implementation and are not likely to be overriding in subclasses.